### DISCUSSION TITLE
buddhism/2021/docs_2021/sadhana/sunset/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#147] [General]
# buddhism/2021/docs_2021/sadhana/sunset/

https://shanenull.com/buddhism/2021/docs_2021/sadhana/sunset/

**Comment by shane0** [comment]
# variation of man in a hole

- he uses the word enlightenment
- man in a hole provides lessons
- but the dialog is a prequel that is good to bad depending on interpretation
- more accurately the professor's intiial delusions only appeared good in reality were bad and shedding delusion can be painful
- happy story shapes do not teach lessons or pertain to survival this is to be or not to be
- this is "not to be"
- tragedy
- this has irony but not satire
- this is a horizontal plot the chaconne bassline is bad - the end
- the villian is the nature of this world
- the dialog is arguing delusion vs reality without avoidance or denial?
- in many of his books he delves into dream realities narrators like the kid the judge anton the father
- many stories questioning if mental fabrication has more substance than the unstable impermanent terminal nature of the world or univers
- stella marris is a psychologist vs a mathmatician
- the passenger is a physicist type man using the world to avoid what he should not have
- child of god is an abandoned man vs a mundane society or god?
- border trilogy is innocence of youth vs the greed of wars or streets
- the bassline is always that the world or the men are coming for you and there is nothing you can do
- in this dialog futility is the pinnacle in blood meridian it is the dance in the counselor it is karma and denial of karma in the border trilogy chasing love to your demise
- the thread that pulls you back is inevitable mortality
- confirming my bias that his books are a form of maranasati

**Reply by shane0** [reply]
# annihilation 

there are other villians a degraded society drugs crime religion probability and mainly education which exposes the cruel nature of reality as a mortal creature a futile mortal creature with no hope but for it to end in annihilation

**Reply by shane0** [reply]
```text
You can acclimate yourself to loss.
You have to.
```

**Reply by shane0** [reply]
> It's that the world is basically a forced labor camp from which the workers, perfectly innocent, are led forth by lottery, a few each day, to be executed. I don't think that's just the way I see it.I think that's the way it is.
>> Are there alternative views? Yes. Will any of them stand close scrutiny? No.
>>> cormac mccarthy - sunset limited

**Comment by shane0** [comment]
https://shanenull.com/stoicism/cormac/

**Comment by shane0** [comment]
```text
When you read the history of the world
you are reading a saga of bloodshed and greed and folly
the import of which is impossible to ignore.
And yet we imagine that the future will somehow be different.
```
